ocs_ci/utility/utils.py

```diff
@@ -4766,14 +4766,126 @@ def get_acm_version():
     from ocs_ci.ocs.resources.csv import get_csvs_start_with_prefix
 
     csv_list = get_csvs_start_with_prefix(
-        "advanced-cluster-management", namespace=constants.ACM_HUB_NAMESPACE
+        "advanced-cluster-management", namespace=namespace
     )
     for csv in csv_list:
         if "advanced-cluster-management" in csv["metadata"]["name"]:
             # extract version string
             return csv["spec"]["version"]
 
 
+def get_dr_hub_operator_version(namespace="openshift"):
```

Comment:
Can we all of those version related functions move to this modul?
https://github.com/red-hat-storage/ocs-ci/blob/master/ocs_ci/utility/version.py

I hope it will also resolve circular dependency so you will not need to have :
```
rom ocs_ci.ocs.resources.csv import get_csvs_start_with_prefix
```

in each of those functions.

Reply:
done

Reply:
the issue reproduced when import `get_csvs_start_with_prefix` function
I moved the new functions to `version.py`
```
 E   ImportError: cannot import name 'get_semantic_version' from partially initialized module 'ocs_ci.utility.version' (most likely due to a circular import) (/home/runner/work/ocs-ci/ocs-ci/ocs_ci/utility/version.py)
```
https://github.com/red-hat-storage/ocs-ci/actions/runs/9598093695/job/26468573010?pr=9956